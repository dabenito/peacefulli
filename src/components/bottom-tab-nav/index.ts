export { Nav } from "./bottom-tab-nav";
