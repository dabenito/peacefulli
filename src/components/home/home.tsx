import React, { PureComponent } from "react";
import { View, Text } from "react-native";
import { styles } from "./style";

export interface HomeProps {}

export interface HomeState {}

export class Home extends PureComponent<HomeProps, HomeState> {
  constructor(props: HomeProps) {
    super(props);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>TODO: Create Home View!</Text>
      </View>
    );
  }
}
