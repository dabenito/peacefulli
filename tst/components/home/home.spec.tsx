import React from "react";
import { render, cleanup } from "@testing-library/react-native";
import { Home } from "../../../src/components/home";

describe("Home Tests", () => {
  afterEach(cleanup);

  // TODO: update this file when implementing home page
  it("loading home page", async () => {
    const home = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
