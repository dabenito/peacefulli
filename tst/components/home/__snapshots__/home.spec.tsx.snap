// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Tests loading home page 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    TODO: Create Home View!
  </Text>
</View>
`;
